middle_end/flambda/types/env/aliases.ml

```diff
@@ -733,7 +780,7 @@ let get_aliases t element =
       Aliases_of_canonical_element.all
         (get_aliases_of_canonical_element t ~canonical_element)
     in
-    Alias_set.create ~canonical_element ~alias_names
+    Alias_set.create ~canonical_element ~coercion_from_canonical_to_element:Coercion.id ~alias_names
```

Comment:
This should fit on 80 columns (there's probably a few other lines that are exceeding this limit in this file)
```suggestion
    Alias_set.create 
      ~canonical_element 
      ~coercion_from_canonical_to_element:Coercion.id 
      ~alias_names
```

Reply:
Fixed (here and in a few other places)